vowpalwabbit/ezexample.h

```diff
@@ -119,7 +119,7 @@ class ezexample
     if (current_ns != 0)
     {
       str[0] = current_ns;
-      current_seed = VW::hash_space(*vw_ref, str);
+      current_seed = static_cast<fid>(VW::hash_space(*vw_ref, str));
```

Comment:
what is `fid`?

Reply:
`fid` is a `uint32_t`. Along this file there are casts from `uint64_t` to `uint32_t` wherever a hash occurs.